components/movementsensor/replay/replay.go

```diff
@@ -54,6 +53,9 @@ var (
 	// initializePropertiesTimeout defines the amount of time we allot to the attempt to initialize Properties.
 	initializePropertiesTimeout = 180 * time.Second
 
+	// tabularDataByFilterTimeout defines the amount of time we allot to the call to TabularDataByFilter.
+	tabularDataByFilterTimeout = 20 * time.Second
```

Comment:
Made `tabularDataByFilterTimeout` a variable so we can change the time in the tests. Increased the value to 20 seconds, since app has a timeout of 15 seconds and we don't want to cancel our context within that time frame.